### PR TITLE
fix(api,ui): bugs resulting from recent service def changes

### PR DESCRIPTION
**Pull request by krancour**
This PR follows up on some changes that were missing in #1561

### CODE REVIEW

File: internal/api/promote_stage_v1alpha1.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func (s *server) PromoteStage(
 	}
 
 	stageName := req.Msg.GetStage()
-	if err := validateFieldNotEmpty("name", stageName); err != nil {
```
Comment (krancour) [inline internal/api/promote_stage_v1alpha1.go:28]: Filed name here was mistakenly _not_ updated in #1561.

File: ui/src/features/project/project-details/project-details.tsx [hunk 2]
```diff
@@ -747,7 +747,7 @@ export const ProjectDetails = () => {
                           };
                           if (promotionType === 'default') {
                             promoteAction({
-                              name: promotingStage?.metadata?.name,
```
Comment (krancour) [inline ui/src/features/project/project-details/project-details.tsx:750]: This should have changed in #1561.

File: ui/src/features/project/project-details/project-details.tsx [hunk 3]
```diff
@@ -821,7 +821,7 @@ export const ProjectDetails = () => {
                                   manualApproveAction({
                                     stage: node.data?.metadata?.name,
                                     project: name,
-                                    id: manuallyApproving
```
Comment (krancour) [inline ui/src/features/project/project-details/project-details.tsx:824]: This should have changed in #1561.

File: ui/src/features/project/project-details/update-freight-alias-modal.tsx [hunk 4]
```diff
@@ -55,8 +55,8 @@ export const UpdateFreightAliasModal = ({ freight, project, onSubmit, hide, ...p
       onOk={handleSubmit(async (data) => {
         await updateAliasAction({
           project,
-          freight: freight?.id || '',
-          alias: data.value || ''
```
Comment (krancour) [inline ui/src/features/project/project-details/update-freight-alias-modal.tsx:59]: Both of these should have changed in #1561.

**Review by rbreeze — APPROVED** [review]
LGTM!

**Pull request merged**